_health-care/_js/components/form-elements/ErrorableRadioButtonGroup.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import _ from 'lodash';
+
+/**
+ * A form radio button group with a label that can display error messages.
+ *
+ * Validation has the following props.
+ * `errorMessage` - Error string to display in the component.
+ *                  When defined, indicates group has a validation error.
+ * `label` - String for the group field label.
+ * `options` - Array of options to populate group.
+ * `required` - boolean. Render marker indicating field is required.
+ * `value` - string. Value of the select field.
+ * `onValueChange` - a function with this prototype: (newValue)
+ */
+class ErrorableRadioButtonGroup extends React.Component {
+  constructor() {
+    super();
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  componentWillMount() {
+    this.inputId = _.uniqueId('errorable-radio-button-group-');
+  }
+
+  handleChange(domEvent) {
+    this.props.onValueChange(domEvent.target.checked);
+  }
+
+  render() {
+    const options = _.isArray(this.props.options) ? this.props.options : [];
+    let reactKey = 0;
+    const optionElements = options.map((obj) => {
+      let label;
+      let value;
+      if (_.isString(obj)) {
+        label = obj;
+        value = obj;
+      } else {
+        label = obj.label;
+        value = obj.value;
+      }
+      const checked = value === this.props.value ? 'checked=true' : '';
+      return (
+        <div key={reactKey++}>
+          <input
+              type="radio"
```

Comment:
The input needs a unique `id` and the label needs a `for` attribute that matches the `id` of the input. This will associate the label with the input so that if someone clicks the label the input will toggle.